common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -93,6 +93,7 @@
                 this.courseSettings = options.courseSettings;
                 this.hideRefineBar = options.hideRefineBar;
                 this.supportsActiveThread = options.supportsActiveThread;
+                this.profilePage = options.profilePage || false;
```

Comment:
Nit: I read this as being a variable containing the profile page. I'd also prefer that this view not know about the existence of the profile page. How about the boolean be called ``showReadState``?